lms/static/sass/shared/_course_object.scss

```diff
@@ -180,78 +191,116 @@
+            content: 'LEARN MORE';
+            position: absolute;
+            width: 200px;
+            height: 50px;
+            z-index: 20;
+            border: 3px solid white;
+            border-radius: 3px;
+            font-family: $sans-serif;
+            font-weight: $font-semibold;
+            font-size: 1.1em;
+            line-height: 44px;
+            color: white;
+            padding: 0 20px;
+            text-align: center;
+            opacity: 0;
+          }
 
           img {
-            display: block;
-            min-height: 100%;
             width: 100%;
           }
         }
+      }
 
-        .desc {
-          @include box-sizing(border-box);
-          height: 120px;
-          overflow: hidden;
-          padding: 10px 10px 12px 10px;
-          position: relative;
-          width: 100%;
+      .course-info {
+        height: 200px;
 
-          section {
-            color: $base-font-color;
-            font: normal 1em/1.6em $serif;
-            margin: 0;
-            height: 100%;
-            overflow: hidden;
-            text-overflow: ellipsis;
-          }
+        .course-organization, .course-code, .course-date {
+          color: #3d3e3f;
+          font-size: 0.9em;
         }
 
-        .bottom {
-          @include box-sizing(border-box);
-          padding: 0px 10px 10px 10px;
-          width: 100%;
+        .course-organization {
+          padding: 11px 15px 2px;
+          line-height: 16px;
+        }
 
-          .university {
-            color: $lighter-base-font-color;
-            letter-spacing: 1px;
-          }
+        .course-code {
+          padding: 0 15px;
+          line-height: 24px;
+        }
 
-          .start-date  {
-            border-left: 1px solid $border-color-2;
-            margin-left: ($baseline/4);
-            padding-left: ($baseline/2);
-            color: $lighter-base-font-color;
-            letter-spacing: 1px;
-          }
+        .course-title {
+          height: 72px;
+          font-size: 1.1em;
+          color: #701f3f;
+          padding: 0 15px;
+          margin: 5px 0;
+          line-height: 24px;
+        }
+
+        .course-date {
+          margin-top: 30px;
+          line-height: 20px;
+          padding: 3px 15px;
         }
       }
 
       &:hover, &:focus {
-        background: $course-profile-bg;
-        border-color: $border-color-1;
-        box-shadow: 0 1px 16px 0 rgba($shadow-color, 0.4);
+        .cover-image {
+          &:before {
+            @include fade-transition-delay(0.3s);
+            opacity: 0.6;
+          }
 
-        .info {
-          top: -150px;
+          &:after {
+            @include fade-transition-delay(0.3s);
+            opacity: 1;
+          }
         }
+      }
+    }
 
-        .meta-info {
-          opacity: 0;
-        }
+    .more-courses {
+      text-align: center;
+      margin-right: 0;
+
+      @include media($desktop) {
+        text-align: right;
```

Comment:
RTL Support (https://github.com/edx/ux-pattern-library/wiki/Styleguide:-Sass-&-CSS#right-to-left-rtl-support) is needed here.